packages/next/src/build/webpack-build/impl.ts

```diff
@@ -329,6 +333,11 @@ export async function webpackBuildImpl(
       duration: webpackBuildEnd[0],
       buildTraceContext: traceEntryPointsPlugin?.buildTraceContext,
       pluginState: getPluginState(),
+      telemetryState: {
```

Comment:
Pass the state from telemetry plugin out from the worker, we introduce a `telemetryState` here